ubuntu_updates_check.sh

```diff
@@ -40,6 +40,8 @@ fi
 
 if [ $((PENDING_OTHER+PENDING_SECURITY)) -gt 0 ]; then
   UPGRADABLE_PACKAGES=$(apt list --upgradable 2>/dev/null | grep -v Listing | awk -F'/' '{print $1}' | paste -sd ',' -)
+else
+  UPGRADABLE_PACKAGES="n/a"
```

Comment:
For no particular reason the slash in the value makes me nervous. I wonder if simply "none" would suffice?